public/app/percona/backup/components/ScheduledBackups/ScheduledBackups.test.tsx

```diff
@@ -1,17 +1,27 @@
 import React from 'react';
-import { Table } from 'app/percona/integrated-alerting/components/Table';
-import { stubs } from './__mocks__/ScheduledBackups.service';
+import { render, screen } from '@testing-library/react';
+import { StoreState } from 'app/types';
+import { configureStore } from 'app/store/configureStore';
+import { Provider } from 'react-redux';
 import { ScheduledBackups } from './ScheduledBackups';
-import { render, waitFor } from '@testing-library/react';
 
 jest.mock('./ScheduledBackups.service');
-jest.mock('app/percona/integrated-alerting/components/Table', () => ({
-  Table: jest.fn(({ children }) => <div data-testid="table">{children}</div>),
-}));
 
 describe('ScheduledBackups', () => {
   it('should send correct data to Table', async () => {
-    await waitFor(() => render(<ScheduledBackups />));
-    expect(Table).toHaveBeenCalledWith(expect.objectContaining({ data: stubs }), expect.anything());
+    render(
+      <Provider
+        store={configureStore({
+          percona: {
+            user: { isAuthorized: true, isPlatformUser: false },
+            settings: { result: { backupEnabled: true, isConnectedToPortal: false } },
+          },
+        } as StoreState)}
+      >
+        <ScheduledBackups />
+      </Provider>
+    );
+    await screen.findByText('Backup 1');
```

Comment:
Indeed. I usually avoid `waitFor`, because it doesn't tell us much. Waiting for `Backup 1` means the data was loaded already on the table, which is what visually tells us the page is done. It's a more "UI" approach of testing, closer to what users see